build-tools/src/main/resources/roles.yml

```diff
@@ -11,3 +11,11 @@ _es_test_root:
     - application: "*"
       privileges: [ "*" ]
       resources: [ "*" ]
+
+_ccs_admin:
```

Comment:
Safe to ignore: just a convenience for local testing -- won't be part of any "real" PRs.